espnet/asr/asr_mix_utils.py

```diff
@@ -27,15 +27,19 @@
 # * -------------------- training iterator related -------------------- *
 def make_batchset(data, batch_size, max_length_in, max_length_out,
                   num_batches=0, min_batch_size=1):
-    """Make batch set from json dictionary
-
-    :param Dict[str, Dict[str, Any]] data: dictionary loaded from data.json
-    :param int batch_size: batch size
-    :param int max_length_in: maximum length of input to decide adaptive batch size
-    :param int max_length_out: maximum length of output to decide adaptive batch size
-    :param int num_batches: # number of batches to use (for debug)
-    :param int min_batch_size: mininum batch size (for multi-gpu)
-    :return: List[Tuple[str, Dict[str, List[Dict[str, Any]]]] list of batches
+    """Make batch set from json dictionary.
+
+    Args:
+        data (dict[str, list[Any]): Dictionary loaded from data.json.
+        batch_size (int): Batch size.
+        max_length_in (int): Maximum length of input to decide adaptive batch size.
+        max_length_out (int): Maximum length of output to decide adaptive batch size.
+        num_batches (int): # Number of batches to use (for debug).
+        min_batch_size (int): Mininum batch size (for multi-gpu).
+
+    Returns:
+        list[tuple[str, dict[str, list[dict[str, Any]]]]: List of batches.
```

Comment:
typo: `list[tuple[str, dict[str, list[dict[str, Any]]]]]` list (1) of tuple (2) of dict (3) of list (4) of dict(5), you need 5 `]`. LOL